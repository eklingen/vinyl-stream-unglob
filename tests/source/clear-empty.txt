@import 'glob/**/*';
@import 'vrazl/**/*';
@import 'glob/**/*';
