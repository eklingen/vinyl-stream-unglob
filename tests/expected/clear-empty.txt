@import 'glob/one.txt'; @import 'glob/two.txt';

@import 'glob/one.txt'; @import 'glob/two.txt';
